tests/fixtures/issues/issue-1253.php

```diff
@@ -0,0 +1,216 @@
+<?php
```

Comment:
Oh, forgot to mention this file.  This is just adding a few Kindle useragents to the test suite to increase coverage of the kindle.json file. I mainly did this as a test of how codecov.io showed a change in coverage from one commit to the next. I felt it was fine to leave in, but can remove and re-submit as a separate PR if desired.